### PR TITLE
Update python.rst

### PR DESCRIPTION
**Pull request by eglerean**
I have read + tested the code and it works. For clarity I have added some extra line of code (e.g. how to test the function and how to test the class).

I have left some "???" for @rkdarst to fill in as I suppose he will be the one covering this chapter.

### CODE REVIEW

**Review by eglerean — COMMENTED** [review]
One typo... maybe?

File: python.rst [hunk 1]
```diff
@@ -153,11 +159,11 @@ statically typed language, types are associated with lvalues, and are
...
+??? (lesson here)
 
 
 
 .. keypoints::
 
-   - K1
-   - K2
+   - K1 Python offers a nice set of basic types as many other programming languages
+   - K2 Python is strongly typed and dynamically typed
```
Comment (rkdarst) [inline python.rst:169]:
```suggestion
   - Python offers a nice set of basic types as many other programming languages
   - Python is strongly typed and dynamically typed
```

File: python.rst [hunk 2]
```diff
@@ -3,13 +3,13 @@ Introduction to Python
 
 .. questions::
 
-   - Q1
-   - Q2
+   - Q1 What are the basic blocks of Python language?
+   - Q2 How are functions and classes defined in Python?
```
Comment (rkdarst) [inline python.rst:7]:
```suggestion
   - What are the basic blocks of Python language?
   - How are functions and classes defined in Python? 
```

File: python.rst [hunk 3]
```diff
@@ -3,13 +3,13 @@ Introduction to Python
...
-   - Q2
+   - Q1 What are the basic blocks of Python language?
+   - Q2 How are functions and classes defined in Python? 
 
 .. objectives::
 
-   - O1
-   - O2
+   - O1 Get a *very* short introduction to Python types and syntax
+   - O2 ???
```
Comment (rkdarst) [inline python.rst:12]:
```suggestion
   - Get a *very* short introduction to Python types and syntax
   - Be able to follow the rest of the examples in the course, even if you don't understand everything perfectly.
```

**Review by rkdarst — APPROVED** [review]
thanks, these are perfect!

**Pull request merged**